pymongo/synchronous/encryption.py

```diff
@@ -707,10 +707,15 @@ def __init__(
         self._io_callbacks: Optional[_EncryptionIO] = _EncryptionIO(
             None, key_vault_coll, None, opts
         )
+
+        # Pass bypass_encryption=True to skip loading crypt_shared.
         self._encryption = ExplicitEncrypter(
             self._io_callbacks,
             _create_mongocrypt_options(
-                kms_providers=kms_providers, schema_map=None, key_expiration_ms=key_expiration_ms
+                kms_providers=kms_providers,
+                schema_map=None,
+                key_expiration_ms=key_expiration_ms,
+                bypass_encryption=True,
```

Comment:
Inline the comment in both places: `bypass_encryption=True,  # Don't load crypt_shared`